cmd/info.go

```diff
@@ -1,36 +1,36 @@
-/*
-Copyright © 2022 NAME HERE <EMAIL ADDRESS>
-
-*/
 package cmd
 
 import (
 	"fmt"
 	"github.com/kubefirst/nebulous/configs"
 	"github.com/spf13/cobra"
 	"log"
+	"runtime"
 )
 
 // infoCmd represents the info command
 var infoCmd = &cobra.Command{
 	Use:   "info",
-	Short: "Provide a general overview of host machine and cli",
-	Long: `Command used to allow a deeper inspection of the host machine 
-	and cli version runnig and its current state. Tool recommended for troubleshooting 
-	installations`,
+	Short: "Provides general Kubefirst setup data",
+	Long:  `Provides machine data, files and folders paths`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Printf("flare-cli golang utility version: v%s \n", NebolousVersion)
-		fmt.Printf("OS type: %s\n", localOs)
-		fmt.Printf("Arch: %s\n", localArchitecture)
-		fmt.Printf("$HOME folder: %s\n", home)
-		fmt.Printf("kubectl used: %s\n", kubectlClientPath)
-		fmt.Printf("terraform used: %s\n", terraformPath)
-		fmt.Printf("Kubeconfig in use: %s\n", kubeconfigPath)
-		err := configs.CheckFlareFile(home)
+
+		config := configs.ReadConfig()
+
```

Comment:
yes, that's the plan, I added this to the PR comment:

`It will have a single instance of Config, that will be passed by parameters for the functions who needs any kind of configuration.`